src/proto/thread_telemetry.proto

```diff
@@ -14,20 +14,19 @@ package threadnetwork;
 // Usage:
 // Delete field: do not directly delete field. Deprecate it instead.
 message TelemetryData {
-  enum NodeType {
-    NODE_TYPE_UNSPECIFIED = 0;
-    NODE_TYPE_ROUTER = 1;
-    NODE_TYPE_END = 2;
-    NODE_TYPE_SLEEPY_END = 3;
-    NODE_TYPE_MINIMAL_END = 4;
+  message Duration {
```

Comment:
This is borrowed from [google](https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/duration.proto).
I wonder whether we need it to be so precise. It probably acceptable to use `uint32_t` representing the duration in seconds.

Reply:
How about to keep this Duration format currently: 1. for age / timeout use case, only the seconds field is populated, so the wired payload size is optimized without the nano field. 2. compatible with the parsing proto logic in upper layer, and extensible if we need to have finer nano granularity in future.